scripts/install

```diff
@@ -114,14 +114,30 @@ app_uid=$(kubectl get "applications.app.k8s.io/$name" \
   --namespace="$namespace" \
   --output=jsonpath='{.metadata.uid}')
 
+# Create ServiceAccount instance.
+deployer_name="$(make_dns1123_name.py --name="${name}")-deployer-sa"
```

Comment:
Can you add a comment that this needs to match the deployer name that is provisioned in provision.py

Reply:
Should the variable name be service_account_name?

Reply:
Instead of trying to coordinate naming schemes, I've changed provision.py to require the caller to pass in the desired deployer service account name (it's fine if this KSA doesn't exist yet).

---

Often times we say 'deployer' to refer to the deployment service account. Changed variable name to `deployer_service_account_name` to be more clear.